src/components/molecules/SocialButtons/GoogleButton/index.tsx

```diff
@@ -0,0 +1,8 @@
+import { isExpoGo, isWeb } from '@baca/constants'
+
+import { NativeGoogleButton } from './NativeGoogleButton'
+
+export const GoogleButton = () => {
+  if (isExpoGo || isWeb) return undefined //TODO: Add google button for web
```

Comment:
Change `undefined` to `null`

```suggestion
  // TODO: Add google button for web
  if (isExpoGo || isWeb) return null 
```